src/main/java/com/palantir/docker/compose/execution/RetryingDockerCompose.java

```diff
@@ -64,6 +64,15 @@ public void rm() throws IOException, InterruptedException {
     }
 
     @Override
+    public void exec(DockerComposeExecOption dockerComposeExecOption, String containerName,
```

Comment:
I win.

Sent from my iPhone

> On 19 Apr 2016, at 11:25, joelea <notifications@github.com> wrote:
> 
> In src/main/java/com/palantir/docker/compose/execution/RetryingDockerCompose.java:
> 
> > @@ -64,6 +64,15 @@ public void rm() throws IOException, InterruptedException {
> >      }
> >  
> >      @Override
> > +    public void exec(DockerComposeExecOption dockerComposeExecOption, String containerName,
> Agreed - I say we make this a non-retryable one for now and see what happens.
> 
> Also, if this gets any more calls, I'm totally making this use annotations and some fancy reflection nonesense.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
>